x/upgrade/keeper/grpc_query_test.go

```diff
@@ -4,40 +4,62 @@ import (
 	gocontext "context"
 	"testing"
 
+	"github.com/stretchr/testify/suite"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
 	"github.com/cosmos/cosmos-sdk/baseapp"
 	"github.com/cosmos/cosmos-sdk/simapp"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
-	"github.com/stretchr/testify/require"
-	abci "github.com/tendermint/tendermint/abci/types"
 )
 
-func TestGRPCQueryUpgrade(t *testing.T) {
+type UpgradeTestSuite struct {
+	suite.Suite
+
+	app         *simapp.SimApp
+	ctx         sdk.Context
+	queryClient types.QueryClient
+}
+
+func (suite *UpgradeTestSuite) SetupTest() {
 	app := simapp.Setup(false)
 	ctx := app.BaseApp.NewContext(false, abci.Header{})
 
 	queryHelper := baseapp.NewQueryServerTestHelper(ctx, app.InterfaceRegistry())
 	types.RegisterQueryServer(queryHelper, app.UpgradeKeeper)
 	queryClient := types.NewQueryClient(queryHelper)
 
-	t.Log("Verify that the scheduled upgrade plan can be queried")
+	suite.app = app
+	suite.ctx = ctx
+	suite.queryClient = queryClient
+}
+
+func (suite *UpgradeTestSuite) TestGRPCQuery() {
+	queryClient := suite.queryClient
+
+	suite.T().Log("Verify that the scheduled upgrade plan can be queried")
 	plan := types.Plan{Name: "test-plan", Height: 5}
-	app.UpgradeKeeper.ScheduleUpgrade(ctx, plan)
+	suite.app.UpgradeKeeper.ScheduleUpgrade(suite.ctx, plan)
 
 	res, err := queryClient.CurrentPlan(gocontext.Background(), &types.QueryCurrentPlanRequest{})
-	require.NoError(t, err)
-	require.Equal(t, res.Plan, &plan)
+	suite.NoError(err)
```

Comment:
ditto for the other checks 👍 
```suggestion
	suite.Require().NoError(err)
```